metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/AlertsUIServiceImpl.java

```diff
@@ -136,12 +136,15 @@ public void saveAlertsUIUserSettings(AlertsUIUserSettings alertsUIUserSettings)
   }
 
   @Override
-  public boolean deleteAlertsUIUserSettings(String user) {
-    boolean success = true;
+  public boolean deleteAlertsUIUserSettings(String user) throws RestException {
+    boolean success = false;
     try {
-      userSettingsClient.delete(user, ALERT_USER_SETTING_TYPE);
+      if(userSettingsClient.findOne(user, ALERT_USER_SETTING_TYPE).isPresent()) {
```

Comment:
This was needed to maintain the behavior of returning false if you attempt to delete settings that don't exist.  When issuing the delete you don't get any indication of whether something was deleted or not.